src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/Utility/ProjectInstalledPackages.cs

```diff
@@ -0,0 +1,27 @@
+using System;
+using NuGet.Packaging.Core;
+using NuGet.Versioning;
+
+namespace NuGet.PackageManagement.VisualStudio.Utility
+{
+    internal class ProjectInstalledPackages
```

Comment:
naming: This represents only 1 package right?

I suggest you rename this to `ProjectInstalledPackage` to better communicate what it represents.